components/SecuritySchemes.yml

```diff
@@ -0,0 +1,38 @@
+openapi: 3.0.0
+info:
+  version: "0.0.3-unstable"
+  title: VC API
+  description: This is an Experimental Open API Specification for the [VC Data Model](https://www.w3.org/TR/vc-data-model/).
+  license:
+    name: W3C Software and Document License
+    url: http://www.w3.org/Consortium/Legal/copyright-software.
+  contact:
+    name: GitHub Source Code
+    url: https://github.com/w3c-ccg/vc-api
+paths:
+components:
+  securitySchemes:
+    noAuth:
+      type: http
+      scheme: bearer
+      bearerFormat: emptyString
+
+    zCap:
+      type: http
+      scheme: signature
+      bearerFormat: AuthorizationCapability
+
+    didAuth:
+      type: http
+      scheme: signature
+      bearerFormat: VerifiablePresentation
+
+    oAuth2:
+      type: oauth2
+      flows:
+        authorizationCode:
```

Comment:
You don't have to use authorizationCode -- need to figure out how to say "this is up to implementers".

Reply:
This has been removed in commit f2c17c0907ee42e5a819dfb858432688500ee20b and is up to the instance to define.